modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/BaseMvStoragesTest.java

```diff
@@ -52,11 +52,15 @@
 import org.apache.ignite.lang.IgniteBiTuple;
 import org.apache.ignite.lang.IgniteException;
 import org.jetbrains.annotations.Nullable;
+import org.junit.jupiter.api.RepeatedTest;
 
 /**
  * Base test for MV storages, contains pojo classes, their descriptor and a marshaller instance.
  */
 public abstract class BaseMvStoragesTest {
+    /** To be used in a loop. {@link RepeatedTest} has a smaller failure rate due to recreating the storage every time. */
+    protected static final int REPEATS = 100;
```

Comment:
What's the reason for moving this constant here? Are you sure that every subclass should use the same value?
Maybe new class should have higher repeat value, because deadlock is harder to reproduce

Reply:
Fix it